.gitignore

```diff
@@ -6,3 +6,8 @@ gradle/*
 .gradle/*
 *.iml
 .idea/*
+.classpath
```

Comment:
This will stop new elements being added to git, but there are already existing files / directories which are tracked which need to be removed.  (e.g. build/) .  Could you update this PR with removals of those?  Details on how to do this are here: https://stackoverflow.com/questions/1274057/making-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore